src/main/java/com/amazon/opendistroforelasticsearch/sql/executor/format/SelectResultSet.java

```diff
@@ -305,10 +319,10 @@ private String getFieldName(Field field) {
          */
         Map<String, Field> fieldMap = fetchFieldMap(query);
         List<Schema.Column> columns = new ArrayList<>();
-        for (String field : fields) {
+        for (String fieldName : fields) {
```

Comment:
np: to be consistent would be good to rename `fields` to `fieldNames`

Reply:
Sure, will do.

Reply:
I just found there is an input argument taken the name. Will rename `fields` to fieldNameList.